database/src/main/postgres/flows/V1.9.8__get_flow_partitionings.sql

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+CREATE OR REPLACE FUNCTION flows.get_flow_partitionings(
+    IN i_flow_id               BIGINT,
+    IN i_limit                 INT DEFAULT 5,
+    IN i_offset                BIGINT DEFAULT 0,
+    OUT status                 INTEGER,
+    OUT status_text            TEXT,
+    OUT id                     BIGINT,
+    OUT partitioning           JSONB,
+    OUT author                 TEXT,
+    OUT has_more               BOOLEAN
```

Comment:
do we also want to care whether a given partitioning was `primary` for the given flow?

Reply:
Little value now, and adds a table to the JOIN.
IMHO not worth it.
Btw, we could add the main flow id to partitioning table (for easier search and one less index).